#List of available control frequencies
# control_frequency_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 MSG_CTRL_0_RATE    = 0 #  Message is not being outputted
	uint8 MSG_CTRL_600_RATE  = 1
	uint8 MSG_CTRL_1200_RATE = 2
	uint8 MSG_CTRL_1800_RATE = 3
	uint8 MSG_CTRL_3600_RATE = 4
	uint8 MSG_CTRL_1000_RATE = 5
	uint8 MSG_CTRL_900_RATE  = 6
	uint8 MSG_CTRL_1500_RATE = 7
